src/lib/hooks/useIsClient.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect, useState } from "react";
+
+export default function useIsClient() {
```

Comment:
Why is something like this needed?

Reply:
It was needed for error handling inside validatepassportScore for hydration sake as this page is client side rendered. However we don't need it anymore since we do not handle error there anymore so this is just leftover i forgot to get rid of :<